xbmc/messaging/ApplicationMessenger.h

```diff
@@ -90,6 +90,7 @@
 
 #define TMSG_SET_VOLUME                   TMSG_MASK_APPLICATION + 36
 #define TMSG_SET_MUTE                     TMSG_MASK_APPLICATION + 37
+#define TMSG_RESTART_RENDERING TMSG_MASK_APPLICATION + 38
```

Comment:
Maybe adjust the indentation?

Reply:
clang-format hook is happy.

Reply:
yeap, other lines also don't follow the identation. It's better for someone to do this later as well as killing the defines (out of scope of this PR though).

Reply:
@fritsch clang formatting is turned off, so it shouldn't complain.

But yeah, it's just nitpicking.